docs/source/conf.py

```diff
@@ -0,0 +1,158 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import re
+import sphinx_rtd_theme
+
+# TODO: get current substra version
+def _get_version():
```

Comment:
Update this to get the version: will be in another PR, we need to put in place the "head version" in Substra compatibility table + do releases on this repo

You can delete this function in the meantime